en/peripherals/rtk_gps_cuav_c-rtk.md

```diff
@@ -1,8 +1,9 @@
 # CUAV C-RTK
 
-The [CUAV C-RTK GPS receiver](http://doc.cuav.net/gps/c-rtk/en/) is a RTK module for mass market launched by CUAV. The complete RTK system consists of at least two C-RTK modules \(one for the base station and the others for the aircraft\). Using RTK, PX4 can get its position with centimetre-level accuracy, which is much more accurate than can be provided by a normal GPS.
+The [CUAV C-RTK GPS receiver](http://doc.cuav.net/gps/c-rtk/en/) is an RTK module for mass market launched by CUAV. A complete RTK system consists of at least two C-RTK modules \(one for the base station and the others for the aircraft\). Using RTK, PX4 can get its position with centimetre-level accuracy, which is much more accurate than can be provided by a normal GPS.
```

Comment:
FYI Note that it is "an RTK" not "a RTK". We use "an" because the word starts with a vowel sound ("AR-Tee-Kay")